### ISSUE TITLE
[Screen reader - Logic Apps - Update connector]: Screen reader does not announce the error message displayed after invoking "update connector" button.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6107]
### Describe the issue 

User Experience:

 Screen reader users will be impacted if screen reader does not announce the error message displayed as users will not get error message information and they will get disoriented.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic App Custom Connector' and select any already created ''Logic App Custom Connector' 
- Press 'Edit' button on the menu bar on top of the overview blade. 
- Navigate all the controls on the General tab 
- Input valid and invalid values on the General tab and Press Continue Button. Verify the errors on the page. 
- Press Update Connector. Verify the errors that appear on the page 
- Verify whether screen reader announces the error message displayed after invoking "update connector" button. 

Actual Result:

 Screen reader does not announce the error message displayed after invoking "update connector" button.

 Observation:

 After invoking the 'update connector' control, focus moves to the close button of error message and screen reader announces close button.

Expected Result:

 Screen reader should announce the error message displayed after invoking "update connector" button. Screen reader should announce the information like "host must be supplied".

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29402287

**Comment by hartra344** [comment]
This is getting assigned to the custom connectors team. Not our code.
